### DISCUSSION TITLE
What is the plugin used for the quick chat example?

### DISCUSSION THREAD

**Discussion by chinleung** [CopilotC-Nvim/CopilotChat.nvim#365] [Q&A] [answered]
I'm looking at the quick chat demo in the readme and I was wondering what is the plugin used to have that input? 🙏🏼 

https://gyazo.com/9b8cbf1d78a19f326282a6520bc9aab0

**Comment by thenbe** [comment, marked as answer]
https://github.com/folke/noice.nvim

**Reply by chinleung** [reply]
Thanks!
